apps/post_monitor/4C_post_monitor.cpp

```diff
@@ -1692,8 +1692,7 @@ FOUR_C_NAMESPACE_CLOSE
  *
  * Note: Works in seriell version only! Requires to read one instance of the discretisation!!
  *
- * \author chfoe
- * \date 11/07
+
```

Comment:
This breaks the style: empty line instead of a leading asterisk. Purely cosmetic, of course.